lib/AliasPlugin.js

```diff
@@ -23,6 +22,9 @@ module.exports = class AliasPlugin {
 	constructor(source, options, target) {
 		this.source = source;
 		this.options = Array.isArray(options) ? options : [options];
+		for (const item of this.options) {
+			item.nameWithSlash = item.name + "/";
```

Comment:
Why we need it?

Reply:
It's just a memoization so we don't do this concatenation on every resolution, the concatenation creates an object that is short lived, creating work for the garbage collector. By creating the concatenated string once, storing it and reusing it we save the memory overhead of creating many times the same temporary objects.